panels/UserPanel.php

```diff
@@ -116,4 +108,36 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns an array containing information about the logged-in user.
+     *
+     * The array should contain two items:
+     * - the model that should be set on [[\yii\widgets\DetailView::model]]
+     * - the array that should be set on [[\yii\widgets\DetailView::attributes]]
+     *
+     * @param IdentityInterface $identity
+     * @return array
+     */
+    protected function identityData(IdentityInterface $identity)
+    {
+        if ($identity instanceof Model) {
+            $data = $identity->getAttributes();
+            $attributes = [];
+
+            foreach ($data as $attribute => &$value) {
```

Comment:
That is just a performance recommendation that the [PHP Inspections (EA Extended)](https://plugins.jetbrains.com/idea/plugin/7622-php-inspections-ea-extended-) PhpStorm plugin recommends – it is better to loop through an array this way (and setting `$value` by reference so PHP isn’t creating a bunch of new variables) rather than looping through `array_keys($data)`.